src/main/java/shop/kkeujeok/kkeujeokbackend/member/api/MemberController.java

```diff
@@ -38,4 +38,10 @@ public RspTemplate<TeamDashboardsAndChallengesResDto> getTeamDashboardsAndChalle
         TeamDashboardsAndChallengesResDto response = myPageService.findTeamDashboardsAndChallenges(email, requestEmail, PageRequest.of(page, size));
         return new RspTemplate<>(HttpStatus.OK, "대시보드와 챌린지 정보 조회", response);
     }
+
+    @GetMapping("/mypage/{memberId}")
+    public RspTemplate<MyPageInfoResDto> getFriendProfileInfo(@PathVariable Long memberId) {
+        MyPageInfoResDto friendProfile = myPageService.findFriendProfile(memberId);
+        return new RspTemplate<>(HttpStatus.OK, "친구 프로필 정보 조회", friendProfile);
+    }
```

Comment:
친구 프로필 조회하는 부분에서 매개 변수 명이 memberId인 것은 친구 프로필을 조회하는지 내 프로필을 조회하는지 오해하기 쉬울 것 같아요. 

```suggestion
    @GetMapping("/mypage/{friendId}")
    public RspTemplate<MyPageInfoResDto> getFriendProfileInfo(@PathVariable Long friendId) {
        MyPageInfoResDto friendProfile = myPageService.findFriendProfile(friendId);
        return new RspTemplate<>(HttpStatus.OK, "친구 프로필 정보 조회", friendProfile);
    }
```

이렇게 바꾸는 것도 좋을 것 같아요!

Reply:
혹시 friend라는 테이블이 없더라도 위와 같이 적는게 좋을까요?